app/controllers/api/v1/users_controller.rb

```diff
@@ -158,9 +158,21 @@ def admin_mail
     AdminMailer.email_all_users(params[:email], params[:subject]).deliver_now
   end
 
+  def got
+    @users = User.all
+    if params[:search]
+      @users = User.search(params[:search]).order("created_at DESC")
```

Comment:
Rather than doing this as a custom `search` method on the User, we can do this as a Ransack query right here in the controller, similar to what you have in the MainController.